platform/commonUI/general/src/directives/MCTSplitPane.js

```diff
@@ -94,13 +94,18 @@ define(
          * @constructor
          */
         function MCTSplitPane($parse, $log, $interval) {
+            var splitPaneNumber = 1;
             function controller($scope, $element, $attrs) {
                 var anchorKey = $attrs.anchor || DEFAULT_ANCHOR,
                     anchor,
                     activeInterval,
                     positionParsed = $parse($attrs.position),
                     position; // Start undefined, until explicitly set
 
+                // create unique alias for every instance of MCTSplitPane
+                var mctSplitPaneAlias = "mctSplitPaneAlias" + splitPaneNumber;
+                splitPaneNumber += 1;
```

Comment:
There's no guarantee that these will always be instantiated in the same order.  We could change the page structure at any time and this would break.

Have this be specified in the template instead of `saveUserWidthPreference`, and then behavior would be decoupled from the order in which components are instantiated.